src/main/java/no/nav/veilarbperson/controller/PersonController.java

```diff
@@ -66,6 +72,25 @@ public boolean tilgangTilBruker(@PathVariable("fodselsnummer") String fodselsnum
         return authService.harLesetilgang(fodselsnummer);
     }
 
+    @GetMapping("/{fodselsnummer}/harNivaa4")
+    public HarLoggetInnRespons harNivaa4(@PathVariable("fodselsnummer") String fodselsnummer) {
+        authService.stoppHvisEksternBruker();
+        authService.sjekkLesetilgang(fodselsnummer);
+
+        String innloggerBrukerIdent = authService.getInnloggerBrukerIdent();
+        UnleashContext context = UnleashContext.builder().userId(innloggerBrukerIdent).build();
```

Comment:
UnleashService setter context for deg, så tror ikke man trenger å sette det med mindre man har spesielle behov

Reply:
Setter ikke med userid. trenger den for feature togle

Reply:
UnleashService defaulter til `resolveUnleashContextFromSubject()` som bruker token subject for `userId`, så tror det skal bli riktig uten.

Reply:
så greit :)